### PR TITLE
Display time in ms instead of seconds

### LINKED ISSUE #485: Display Time in ms instead of seconds
Hi,
is there a way so display the time in miliseconds instead of seconds?
i think the display in seconds is not very well chosen, normally all pings etc. are in ms

### PR DESCRIPTION
**Pull request by TimZ99**
Mostly templated based. Only backend change is to the history graph.
There are no changes to the database.

Resolves #485.

### CODE REVIEW

File: src/templates/default/module/server/server/view.tpl.html [hunk 1]
```diff
@@ -51,7 +51,7 @@
 				<li class="list-group-item">
 					<dl class="row">
 						<dt class="col-md-4">{{ label_rtime }}:</dt>
-						<dd class="col-md-8">{{ rtime }} {{ label_seconds }}</dd>
+						<dd class="col-md-8">{{ rtime * 1000 }} {{ label_seconds }}</dd>
```
Comment (mateuszmalek) [inline src/templates/default/module/server/server/view.tpl.html]: It should be a new label - `{{ label_miliseconds }}`. As for now it's displaying miliseconds value with 'seconds' label - which is of course invalid :)

**Comment by mateuszmalek** [comment]
General question: what about graphs? Shouldn't they be considered too?

File: src/templates/default/module/server/server/view.tpl.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
 				<li class="list-group-item">
 					<dl class="row">
 						<dt class="col-md-6">{{ label_timeout }}:</dt>
-						<dd class="col-md-6">{{ timeout }} {{ label_seconds }}</dd>
+						<dd class="col-md-6">{{ timeout }} {{ label_milliseconds }}</dd>
```
Comment (mateuszmalek) [inline src/templates/default/module/server/server/view.tpl.html]:
```suggestion
						<dd class="col-md-6">{{ timeout }} {{ label_seconds }}</dd>
```
`{{ label_seconds }}` - timeout should be still in seconds (as in configuration area).

File: src/psm/Module/Server/Controller/ServerController.php [hunk 3]
```diff
@@ -597,7 +597,7 @@ protected function getLabels()
             'label_yes' => psm_get_lang('system', 'yes'),
             'label_no' => psm_get_lang('system', 'no'),
             'label_add_new' => psm_get_lang('system', 'add_new'),
-            'label_seconds' => psm_get_lang('config', 'seconds'),
```
Comment (mateuszmalek) [inline src/psm/Module/Server/Controller/ServerController.php:600]: This line is still required (timeout is in seconds).

File: src/templates/default/module/server/server/view.tpl.html (hunk 2 above)
Comment (TimZ99) [reply src/templates/default/module/server/server/view.tpl.html]: I noticed that too 😅

File: src/templates/default/module/server/history.tpl.html [hunk 4]
```diff
@@ -30,7 +30,7 @@
 		data: {
 			datasets: [
 			{
-				data: [{{ graph.lines.offline.value }}],
+				data: [{{ graph.lines.offline.value * 1000 }}],
```
Comment (mateuszmalek) [inline src/templates/default/module/server/history.tpl.html]: That won't work as `graph.lines.offline.value` is a string!
Changes needs to be made in HistoryGraph class:
`$prev['latency']` as well as `round((float) $record[$key] * 1000, 4) . '}';` (`$record[$key]`) needs to be multiplied. Same situation with `$data['latency_avg']`.
Of course those changes need apropriate actions with labels (like Latency (average) unit).

**Review by mateuszmalek — APPROVED** [review]
Everything looks fine 👍

**Comment by apio-sys** [comment]
Many thanks for this. Works perfect for me.

Joris.

**Pull request merged**